dev-util/sysprof/sysprof-48.0.ebuild

```diff
@@ -0,0 +1,111 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils greadme meson systemd xdg
+
+DESCRIPTION="System-wide Linux Profiler"
+HOMEPAGE="http://sysprof.com/"
+
+LICENSE="GPL-3+ GPL-2+"
+API_VERSION="4"
+SLOT="0/${API_VERSION}"
+KEYWORDS="~amd64 ~arm64 ~loong ~x86"
+IUSE="gtk systemd test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-libs/glib-2.76.0:2
+	gtk? (
+		>=gui-libs/gtk-4.15:4
+		>=gui-libs/libadwaita-1.6_alpha:1
+		x11-libs/cairo
+		x11-libs/pango
+	)
+	systemd? ( sys-apps/systemd )
+	dev-libs/json-glib
+	>=dev-libs/libdex-0.6
```

Comment:
Hello,

Running pkgdiff-mg --build-system sysprof-47.2-r1.ebuild sysprof-48.0.ebuild I saw that libdex dep was updated to 0.9